it/src/main/java/org/corfudb/universe/scenario/config/ScenarioConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.corfudb.universe.scenario.config;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.google.common.collect.ImmutableList;
+import lombok.Data;
+import org.corfudb.universe.scenario.action.Action.AbstractAction;
+import org.corfudb.universe.scenario.spec.Spec;
+
+/**
+ * Represents a scenario config file written in scenario json file which is replayed by a ScenarioTest
+ */
+@Data
+public class ScenarioConfig {
+    private ImmutableList<ScenarioTestConfig> scenario;
+
+    @Data
+    public static class ScenarioTestConfig {
+        private String description;
+        @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "type")
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure using this Jackson deserialization configuration is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4544)